junction/conferences/models.py

```diff
@@ -95,6 +97,19 @@ def is_accepting_proposals(self):
             return False
         return self.proposal_types.filter(end_date__gt=now()).exists()
 
+    @staticmethod
+    def get_year_wise():
+
+        data = Conference.objects.filter(deleted=False)
+        f_data = {}
+
+        for d in data:
+            year = d.start_date.year
+            if year not in f_data:
+                f_data[year] = []
+            f_data[year].append(d)
+        return f_data
```

Comment:
Grouping conferences by year is not the correct approach. Let's say if the Event entry was created in December 2020 for a conference in 2021, we can't group this Event to the year 2020.

What we should have is a Field which acts as the grouping parameter for the events.

Reply:
@palnabarun `start_date` and `end_date` are separate fields which the Admin has to enter while creating the conference entry. It's not the `created_date` of the conference. Though there is another fundamental issue that there can be multiple conferences in one year with multiple CFPs so we should have a conference grouping parameter.

The idea of multiple CPFs in one Conference is fairly recent I believe. Since there has been need of more type of Sessions like Posters.

The original idea of tackling this issue was through Talk types where there was Talks, Workshops received in same CFP. But then organisers wanted more visibility for Different CFPs since they were announced separately.

We can still use Talk Types and keep the description updated for the various types of Tracks, the downside can be the editorial can be long. Let me know your opinion @pradyunsg @palnabarun

Reply:
IMHO, we can have two ways to tackle it:
1. Do nothing and use the talk type attribute to segregate. Cons: Editorial can belong.
2. Decouple Conference and CFP altogether. So, that a conference can have multiple CFP's. Cons: Need to be thought through and designed well for the future.